source/adressering-av-undertegner.rst

```diff
@@ -2,76 +2,106 @@
 
 Adressering av undertegner
 ***************************
-Måten undertegner adresseres og identifiseres på, er litt forskjellig avhengig av om han signerer i direkteflyt eller portalflyt, og om avsender bruker API eller avsenderportalen.
+Undertegner kan adresseres og identifiseres på tre forskjellige måter; 
 
-..  TIP::
-    Hvis du ønsker å lese en introduksjon til de forskjellige flytene en undertegner kan signere i, se :ref:`signeringsflyt`.
+1. Med fødselsnummer
+2. Med e-post og/eller mobilnummer 
+3. Med egenvalgt identifikator 
 
-..  CAUTION::
-    Måten du adresserer undertegner på, påvirker :ref:`innholdet og utseendet på undertegners signatur <identifisereUndertegnere>`.
+Adresseingsalternativene varierer ut i fra om avsender er en offentlig eller privat virksomhet, om dokumentet sendes via API, eller web-portal og hvilken :ref:`signeringsflyt` som skal benyttes.
 
-Posten signering sender ut varsel om signeringsoppdrag til undertegner på e-post og/eller SMS, hvis signering skjer i portalflyt. Ved signering i direkteflyt er det avsender som står for varsling til undertegner.
 
-**Avsender som bruker API kan**
++-----------------+-----------------+---------------------+---------------------------+----------------------+
+|                 |                 |                                                                        |
+|                 |                 |   Adressering / identifisering av undertegner                          |
++-----------------+-----------------+---------------------+---------------------------+----------------------+
```

Comment:
Generelt så er vel "adressering" egentlig bare et konsept i portalflyt, da her er det snakk om at vi kontakter vedkommende som adresseres. I direkteflyt er allerede undertegner "i flyt" når oppdrag opprettes, og fødselsnummer/egenvalgt ID er mer en identifikator som brukes i denne prosessen for å holde undertegnere fra hverandre, og i tillegg er det knyttet litt UX/sikkerhet dersom man bruker fnr som identifikator her ved at man sikrer at det kun er mulig for person med angitt fnr å signere.

Ser at du har "Adressering / identifisering" i overskrift her, men i mitt hode er ikke dette helt de samme konseptene. Det er absolutt mulig at det er greit å kontekst av denne tabellen, og informasjonen vi ønsker å kommunisere her altså, men tenkte å bare nevne det.